dockerfiles/build.sh

```diff
@@ -1,12 +1,17 @@
 #!/bin/sh
 
 set -ex
-
+if [[ -z ${DOCKER} ]]; then
+    command -v podman >/dev/null 2>&1 && DOCKER=podman
+fi
+if [[ -z ${DOCKER} ]]; then
+    command -v docker >/dev/null 2>&1 && DOCKER=docker
+fi
```

Comment:
I also favor podman over docker. But in order not to change the default behavior of the scripts consider swapping these two if's